pkg/config/containers.conf

```diff
@@ -341,6 +341,15 @@ default_sysctls = [
 #
 #events_logger = "journald"
 
+# A is a list of directories which are used to search for helper binaries.
+#
+#helper_binaries_dir = [
+#  "/usr/local/libexec/podman",
```

Comment:
Indentation is off in the first entry.

Reply:
This is intentional, check the other config options in the file.